app/views/pets/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_tag "/shelters/#{@shelter.id}/pets" do %>
```

Comment:
I found that with these form tags, it defaults to post method, which my tests showed me